src/components/Events/Events.jsx

```diff
@@ -156,7 +168,7 @@ class Events extends React.Component {
         position: 'absolute', top: '50%', left: '50%', transform: 'translate(-50%, -50%)',
       }}
       >
-PLEASE WAIT... LOADING SOME SERIOUS BULLSHIT
+        <img src="/images/loader.gif" alt="loader" style={{ width: '500px', height: 'auto' }} />
```

Comment:
I liked the earlier version :P

Do you think the loader.gif is little too large 1.35MB?